src/App.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.PureComponent<State> {
+  state: State = {
+    pressedKey: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleKeySet);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeySet);
+  }
+
+  handleKeySet = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  handleKeyUnset = () => {
+    this.setState({ pressedKey: '' });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        {this.state.pressedKey ? (
+          <p
+            className="App__message"
+          >
+            The last pressed key is [
```

Comment:
It is better to use interpolation instead of writing it in this way. Now you are connecting some HTML text with JS variable, as far as I know it is not a good practice)
`{`The last pressed key is ${this.state.pressedKey}`}`